src/main/java/edu/cornell/kfs/fp/batch/service/impl/CuDistributionOfIncomeAndExpenseDocumentGenerator.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.fp.batch.service.impl;
+
+import java.util.function.Supplier;
+
+import org.kuali.kfs.krad.bo.AdHocRoutePerson;
+import org.kuali.kfs.krad.bo.Note;
+
+import edu.cornell.kfs.fp.document.CuDistributionOfIncomeAndExpenseDocument;
+
+public class CuDistributionOfIncomeAndExpenseDocumentGenerator
+        extends AccountingDocumentGeneratorBase<CuDistributionOfIncomeAndExpenseDocument> {
+
+    public CuDistributionOfIncomeAndExpenseDocumentGenerator() {
+        super();
+    }
+
+    public CuDistributionOfIncomeAndExpenseDocumentGenerator(
+            Supplier<Note> bareNoteGenerator, Supplier<AdHocRoutePerson> bareAdHocRoutePersonGenerator) {

```

Comment:
I like the "empty" naming convention versus "bare" .... should these be changed to that convention as well?

Reply:
Yes, I'll change the names on these as well. Thanks for spotting them!